src/main/java/hudson/plugins/ec2/EC2AbstractSlave.java

```diff
@@ -92,7 +82,6 @@
     public final String jvmopts; // e.g. -Xmx1g
     public final boolean stopOnTerminate;
     public final String idleTerminationMinutes;
-    public final boolean useDedicatedTenancy;
```

Comment:
You cannot simply remove a field, this can cause deserialization issues make that field transient if you do not need it, its readResolve() method should do the appropriate migration

Reply:
Thanks @res0nance , Added the review comments.